app/Libraries/Discord.php

```diff
@@ -26,6 +30,7 @@ public function __construct()
         $this->token = config('services.discord.token');
         $this->guild_id = config('services.discord.guild_id');
         $this->base_url = config('services.discord.base_discord_uri').'/guilds';
+        $this->base_user_url = config('services.discord.base_discord_uri').'/users';
```

Comment:
I'd prefer if we amended the `$this->base_url` to remove the `/guilds` bit and then update the class elsewhere to handle that. Setting the base URL as `/guilds` in hindsight. We can then remove the `$this->base_user_url`.

Reply:
Done! https://github.com/VATSIM-UK/core/pull/2029/commits/bacf74390e917a543d3d37b3513e289f79bdac1a